MathLib/Integration/GaussLegendreTet.cpp

```diff
@@ -24,37 +25,42 @@ GaussLegendreTet<2>::X = {{ {{1./4., 1./4., 1./4.}},
                             {{1./6., 1./6., 1./2.}} }};
 double const GaussLegendreTet<2>::W[5] = {-2./15., 0.075, 0.075, 0.075, 0.075};
 
+constexpr std::array<std::array<double, 3>, GaussLegendreTet<3>::NPoints>
```

Comment:
This is not properly implemented on the MSVC. Either remove this, or remove the definitions `double const a = xxx.xxx` below. Or maybe the definitions must be costexpr too....

Reply:
But `double const a = xxx.xxx` is already constexpr, isn't it?
I'll revert the last commit. I hope that helps.

Reply:
> /var/jenkins/workspace/4-52JWG6MV4D/ogs/MathLib/Integration/GaussLegendreTet.cpp:55:1: error: body of constexpr function 'constexpr std::array<std::array<double, 3ul>, 14ul> MathLib::initGLTet3X()' not a return-statement

Probably also envinf's compiler has problems recognizing the constexprness of this "complicated" funciton.